### ISSUE TITLE
Missing GISTIC data for SKCM

### ISSUE THREAD

**Issue by pcheng84** [waldronlab/curatedTCGAData#29]
Hi Marcel and Levi,

I noticed some of the cancers were missing the GISTIC data like SKCM and LAML. I was wondering when these will be uploaded>

> curatedTCGAData("SKCM", "GISTIC*", FALSE)
Error in curatedTCGAData("SKCM", "GISTIC*", FALSE) : 
  Cancer and data type combination(s) not available

Thanks,
Phil

**Comment by LiNk-NY** [comment]
Hi Phil, @pcheng84 

It looks like `RTCGAToolbox` only serves `Peaks` data for those cancer types. 
There might be a bug somewhere in the code. Can you provide a link to the dataset for download from `gdac.broadinstitute.org`? 
It could be that `getFirehoseData` is not fetching the right URL for these cancer types. 

Best,
Marcel

**Comment by pcheng84** [comment]
Here is the link for SKCM
http://gdac.broadinstitute.org/runs/analyses__2016_01_28/data/SKCM-TM/20160128/gdac.broadinstitute.org_SKCM-TM.CopyNumber_Gistic2.Level_4.2016012800.0.0.tar.gz

and for LAML
http://gdac.broadinstitute.org/runs/analyses__2016_01_28/data/LAML-TB/20160128/gdac.broadinstitute.org_LAML-TB.CopyNumber_Gistic2.Level_4.2016012800.0.0.tar.gz

**Comment by LiNk-NY** [comment]
Thanks @pcheng84 , I'll send in a patch through `RTCGAToolbox` this week. More info to come.

**Comment by LiNk-NY** [comment]
Hi Phil, @pcheng84 
I've sent in the patch to `RTCGAToolbox` and can be seen here: https://github.com/LiNk-NY/RTCGAToolbox/commit/cf8cad1111ed0d285467d5927195bb4121bcc129

It will take some time for it to show up on `curatedTCGAData` since it's part of a pipeline.
Entries will have to be added to `ExperimentHub` for these datasets. 

For now, you can use `RTCGAToolbox::biocExtract` to get these in somewhat workable shape. 

```r
library(RTCGAToolbox)
sk <- getFirehoseData("SKCM", GISTIC = TRUE)
GIST <- biocExtract(sk, "GISTIC")
```

Best, 
Marcel

**Comment by pcheng84** [comment]
Hi Marcel,

Great! Thanks for the hotfix!

Cheers,
Phil

**Comment by LiNk-NY** [comment]
Hi Phil, @pcheng84 

This has been resolved in version `1.5.11`. Or the Release version to come out soon `1.6.0`. 
In any event, you can use the GitHub version to reliably obtain `LAML` and `SKCM` GISTIC datasets.
Thanks!

Marcel
